docs/style-guide.rst

```diff
@@ -143,10 +143,10 @@ No::
     }
 
 
-    import "owned";
+    import "./Owned.sol";
```

Comment:
I don't think `./` is necessary.

Reply:
@axic I've pushed a demo of the code to this repository https://github.com/ltfschoen/dao

If you follow the steps in the Readme by installing relevant dependencies:
```bash
npm install -g truffle ganache-cli
```
Then starting Ganache CLI with:
```bash
ganache-cli \
  --port="8500" \
  --mnemonic "copy obey episode awake damp vacant protect hold wish primary travel shy" \
  --verbose \
  --networkId=3 \
  --gasLimit=7984452 \
  --gasPrice=2000000000;
```
Then in a separate terminal tab compile and deploy the contracts by running the following:
```bash
$ truffle migrate --network development
Compiling ./contracts/Congress.sol...
Compiling ./contracts/Migrations.sol...
Compiling ./contracts/Owned.sol...
Writing artifacts to ./build/contracts

Using network 'development'.

Running migration: 1_initial_migration.js
  Deploying Migrations...
  ... 0x39217f6841bbdaef54fc759d3a6ff2b4e6455eaa02e417839c4d5826bb696129
  Migrations: 0x5b80e4653adbc5e8e2387474d4de0eeed34779ea
Saving successful migration to network...
  ... 0x910366905fa49da62c436bbb01f32f676063ed63e4988fad270122c625dd3192
Saving artifacts...
Running migration: 2_deploy_contracts.js
  Deploying Owned...
  ... 0x9d76b243b694da1765146293224ce2d100935692e66428a54e868c52a875edaa
  Owned: 0x4cdb6ac6c502d9dca1a0b70fdc1cd7e64f89e1bf
  Deploying Congress...
  ... 0xb58972d9d5a29ec0fa7fe85b9c6751c050888566e3be9c1a927447f310d96f76
  Congress: 0xdebc221ddbf0777c57c6369d03ee875ee8a4f694
Saving successful migration to network...
  ... 0xed84439c98e43d1e5fba16ff6fb7e204f7edfd776cb9a237bc364878275d178d
Saving artifacts...
```

If you then try and omit the the `./` (which indicates that the contract file being imported is located in the same directory relative to the contract file that is importing it) and change it from `import "./Owned.sol";` to `import "Owned.sol";` on [Line 3 of Congress.sol](https://github.com/ltfschoen/dao/blob/master/contracts/Congress.sol#L3), and then restart Ganache CLI, remove the build directory `rm -rf build/`, and then try to compile and migrate again, then the output is the following:
```bash
$ truffle migrate --network development
Compiling ./contracts/Congress.sol...
Compiling ./contracts/Migrations.sol...
Compiling ./contracts/Owned.sol...
Compiling Owned.sol...

Invalid input source specified.
Compilation failed. See above.
```

Reply:
Oh I would actually say that is a shortcoming of truffle as opposed to the compiler :)

Reply:
I would actually always recommend using `./`. If you do not use it, you are essentially referring to `/file`. I don't think it is a shortcoming of truffle.

Reply:
Note that if you omit the preceding dot and use just `import "/Owned.sol";` then you'll get the same error `Invalid input source specified. Compilation failed.` as you get when you use just `import "Owned.sol";`